packages/integrations/vercel/README.md

```diff
@@ -175,6 +175,8 @@ You can use Vercel middleware to intercept a request and redirect before sending
     ```
 1. While developing locally, you can run `vercel dev` to run middleware. In production, Vercel will handle this for you.
 
+:::caution[Trying to rewrite?] Currently rewriting a request with middleware only works for static files. :::
```

Comment:
```suggestion
> **Warning**
> Trying to rewrite? Currently rewriting a request with middleware only works for static files.
```
I think because this README will also be displayed on GitHub, we need to use their syntax here, and probably lose the fancy title. Otherwise, Sarah is cool with this note!